Makefile

```diff
@@ -32,6 +32,14 @@ clean: ## Removes binary, cache folder and docker images
 	@rm -rf ${BINARY_DIR}
 	@rm -rf $(PLUGIN_DEPLOYMENTS_DIR)
 
+.PHONY: tools
+tools: ## Installs required go tools
```

Comment:
is this target called by default or do I need to call it separately? I don't see the target as part of the build or any other target.
Same for the target "imports"

Reply:
There is no need to run `tools` once you get it set up (e.g. `setup.sh`). I would think about it as a separate/isolated step. In fact we could re-use it as well in `.travis.yml`.

The same for `imports` - we could make it part of the `build` target but this will result in adding more changes to what you have done on your own so that might be a problem (mainly b/c of the aforementioned `import` removal).